ui/app/services/flags.ts

```diff
@@ -31,15 +31,15 @@ export default class flagsService extends Service {
     this.flags = flags;
   }
 
-  get managedNamespaceRoot() {
-    if (this.flags && this.flags.includes(FLAGS.vaultCloudNamespace)) {
-      return 'admin';
-    }
-    return null;
+  get isManaged(): boolean {
+    return this.flags && this.flags.includes(FLAGS.vaultCloudNamespace);
+  }
+
+  get managedNamespaceRoot(): string | null {
```

Comment:
out of curiosity - is declaring the return here `: string | null` necessary? It _seems_ like typescript should be able to infer the return of this getter since it's fairly straight forward

Reply:
@noelledaley I'm going to punt this question to you. I probably have the least amount of typescript knowledge on the team. Do you know if this is just a best practice to keep here?

Reply:
good question! whether or not to explicitly include the return type is somewhat of a stylistic choice, and depends how defensive we want our code to be. 🙂 

examples of why you might include the return type:
- makes the return type more obvious (i.e., no need to hover over the function in VSCode or skim through the code itself)
- enforces type safety. if we later refactor this function to return `undefined` instead of `null`, for example, TS will cause an error and we'll have to decide whether the function should return `string | undefined | null` or _just_ `string | null`. it helps us make more intentional decisions.
- removing the need to hover to find the return type is good for developer accessibility. check out this [tremendously awesome article](https://ashleemboyer.com/blog/how-typescript-can-improve-developer-accessibility) by @ashleemboyer for more.

why you might not include it:
- you're fine fine hovering/reading the code to figure out the return type
- you trust TypeScript's ability to infer the correct type

personally, thus far i haven't included return types in functions this small/simple, but as you can see there _are_ benefits to doing so. so, it's up to you!🌟

Reply:
Thank you so much Noelle for the context/conversation. I'll leave in this case per personal preference.